src/schema/schema_manager.py

```diff
@@ -1261,8 +1261,9 @@ def create_sennet_ids(normalized_class, json_data_dict, user_token, user_info_di
     logger.info("======create_sennet_ids() json_to_post to uuid-api======")
     logger.info(json_to_post)
 
+    uuid_url = _uuid_api_url + "/uuid"
     # Disable ssl certificate verification
-    response = requests.post(url = _uuid_api_url, headers = request_headers, json = json_to_post, verify = False, params = query_parms) 
+    response = requests.post(url = uuid_url, headers = request_headers, json = json_to_post, verify = False, params = query_parms) 
```

Comment:
Also need to add the `/uuid/` path to the GET call: https://github.com/sennetconsortium/entity-api/blob/shirey/entity-support/src/schema/schema_manager.py#L1104